health/health.go

```diff
@@ -48,20 +42,21 @@ var exitChannel = make(chan bool)
 var lock = &sync.RWMutex{}
 
 func init() {
-	foreverHealthy := &healthStatus{
+	foreverHealthy := &domain.HealthCheckStatus{
 		Status:    "passed",
 		Timestamp: time.Now().UTC().Unix(),
 		Interval:  3.156e9, // One century in seconds.
 	}
-	healthStatuses["isvc-internalservices"] = map[string]map[string]*healthStatus{"0": {"alive": foreverHealthy}}
-	healthStatuses["isvc-elasticsearch-logstash"] = map[string]map[string]*healthStatus{"0": {"alive": foreverHealthy}}
-	healthStatuses["isvc-elasticsearch-serviced"] = map[string]map[string]*healthStatus{"0": {"alive": foreverHealthy}}
-	healthStatuses["isvc-zookeeper"] = map[string]map[string]*healthStatus{"0": {"alive": foreverHealthy}}
-	healthStatuses["isvc-opentsdb"] = map[string]map[string]*healthStatus{"0": {"alive": foreverHealthy}}
-	healthStatuses["isvc-logstash"] = map[string]map[string]*healthStatus{"0": {"alive": foreverHealthy}}
-	healthStatuses["isvc-celery"] = map[string]map[string]*healthStatus{"0": {"alive": foreverHealthy}}
-	healthStatuses["isvc-dockerRegistry"] = map[string]map[string]*healthStatus{"0": {"alive": foreverHealthy}}
 
+	// FIXME: These values should be replaced with those maintained in isvcs.IService
```

Comment:
We don't need to fix it for manual HA, but I was going to add a Jira for it as these are only used by the UI right now